tests/optimizer/eliminateDeadGlobals-output.js

```diff
@@ -17,19 +17,6 @@ Module["asm"] = (function(global, env, buffer) {
  var _memcpy = env._memcpy;
  var __ZdaPv = env.__ZdaPv;
  var __ZdlPv = env.__ZdlPv;
- function _emscripten_replace_memory(newBuffer) {
```

Comment:
I don't think we need to remove this from the tests, unless you think it's potentially confusing?

Reply:
Check, restored back. Originally I just search-replace removed all occurrences, but certainly doesn't matter to leave this here.